protocol/version.go

```diff
@@ -9,8 +9,10 @@ const (
 	Version36
 	Version37
 	Version38
+	Version39
 	VersionWhatever    VersionNumber = 0 // for when the version doesn't matter
 	VersionUnsupported VersionNumber = -1
+	VersionUnknown     VersionNumber = -2
```

Comment:
@lucas-clemente: What do you want to do about this? I'd like to merge this PR soon.